velox/connectors/hive/tests/HivePartitionFunctionTest.cpp

```diff
@@ -83,3 +83,101 @@ TEST_F(HivePartitionFunctionTest, bool) {
   assertPartitions(values, 500, {0, 1, 0, 0, 1});
   assertPartitions(values, 997, {0, 1, 0, 0, 1});
 }
+
+TEST_F(HivePartitionFunctionTest, int8_t) {
```

Comment:
naming: int8_t -> tinyint

Reply:
Done. I also rename the existing tests according to this:
- `int64` -> `bigint`
- `string` -> `varchar`
- `bool` -> `boolean`